apps/prairielearn/src/schemas/schemas/infoCourse.json

```diff
@@ -6,23 +6,21 @@
   "additionalProperties": false,
   "required": ["uuid", "name", "title", "topics"],
   "properties": {
+    "title": {
+      "description": "The course title (e.g., 'Introductory Dynamics').",
+      "type": "string"
+    },
     "comment": {
       "description": "Arbitrary comment for reference purposes.",
-      "anyOf": [{ "type": "string" }, { "type": "array" }, { "type": "object" }]
+      "$ref": "#/definitions/CommentJsonSchema"
     },
+    "exampleCourse": { "description": "DEPRECATED -- do not use.", "type": "boolean" },
```

Comment:
This seems to be a newly-added property; is that intentional?

Reply:
Yes. This was missing in the original schema, but we want this.

Reply:
Huh, based on what? I'm curious, no one should have ever used this.

Reply:
Sorry, you are right. I was thinking of that check that you highlighted, but I was able to move that logic out, so that it doesn't need to be apart of any type annotations in ccdc567d0.